src/CakeAdapter.php

```diff
@@ -0,0 +1,574 @@
+     */
+    public function setOptions(array $options)
+    {
+        return $this->_adapter->setOptions($options);
+    }
+
+    /**
+     * Get all adapter options.
+     *
+     * @return array
+     */
+    public function getOptions()
+    {
+        return $this->_adapter->getOptions();
+    }
+
+    /**
+     * Check if an option has been set.
+     *
+     * @param  string $name
+     * @return boolean
+     */
+    public function hasOption($name)
+    {
+        return $this->_adapter->hasOption($name);
+    }
+
+    /**
+     * Get a single adapter option, or null if the option does not exist.
+     *
+     * @param  string $name
+     * @return mixed
+     */
+    public function getOption($name)
+    {
+        return $this->_adapter->getOption($name);
+    }
+
+    /**
+     * Sets the console output.
+     *
+     * @param OutputInterface $output Output
+     * @return AdapterInterface
+     */
+    public function setOutput(OutputInterface $output)
+    {
+        return $this->_adapter->setOutput($output);
+    }
+
+    /**
+     * Gets the console output.
+     *
+     * @return OutputInterface
+     */
+    public function getOutput()
+    {
+        return $this->_adapter->getOutput();
+    }
+
+    /**
+     * Records a migration being run.
+     *
+     * @param MigrationInterface $migration Migration
+     * @param string $direction Direction
+     * @param int $startTime Start Time
+     * @param int $endTime End Time
+     * @return AdapterInterface
+     */
+    public function migrated(MigrationInterface $migration, $direction, $startTime, $endTime)
+    {
+        return $this->_adapter->migrated($migration, $direction, $startTime, $endTime);
+    }
+
+    /**
+     * Does the schema table exist?
+     *
+     * @deprecated use hasTable instead.
+     * @return boolean
+     */
+    public function hasSchemaTable()
+    {
+        return $this->_adapter->hasSchemaTable();
+    }
+
+    /**
+     * Creates the schema table.
+     *
+     * @return void
+     */
+    public function createSchemaTable()
+    {
+        return $this->_adapter->createSchemaTable();
+    }
+
+    /**
+     * Returns the adapter type.
+     *
+     * @return string
+     */
+    public function getAdapterType()
+    {
+        return $this->_adapter->getAdapterType();
+    }
+
+    /**
+     * Initializes the database connection.
+     *
+     * @throws \RuntimeException When the requested database driver is not installed.
+     * @return void
+     */
+    public function connect()
+    {
+        return $this->_adapter->connect();
+    }
+
+    /**
+     * Closes the database connection.
+     *
+     * @return void
+     */
+    public function disconnect()
+    {
+        return $this->_adapter->disconnect();
+    }
+
+    /**
+     * Does the adapter support transactions?
+     *
+     * @return boolean
+     */
+    public function hasTransactions()
+    {
+        return $this->_adapter->hasTransactions();
```

Comment:
No, this means whether or not the adapter supports transactions. We don't really have a method for that